src/display/annotation_layer.js

```diff
@@ -3153,13 +3158,18 @@ class AnnotationLayer {
       }
     }
 
-    this.div.append(element);
-    this.#accessibilityManager?.moveElementInDOM(
-      this.div,
-      element,
-      contentElement,
-      /* isRemovable = */ false
-    );
+    if (popupElements) {
+      // Set the popup just after the first element associated with the popup.
+      popupElements[0].container.after(element);
```

Comment:
Given that the name `popupElements` implies that we can have multiple, is using `[0]` here always correct or should we place all of them after `element` in case there are multiple?